test/lit.cfg

```diff
@@ -641,6 +641,11 @@ if run_vendor == 'apple':
                 "%s %%s" % (target_run_base))
             config.target_run_stdlib_swift = (
                 "%s -Xfrontend -disable-access-control %%s" % (target_run_base))
+            config.target_run_simple_swiftgyb = (
+                'rm -rf %%t && mkdir -p %%t && '
+                '%%gyb %%s -o %%t/main.swift && '
+                '%%line-directive %%t/main.swift -- %s %%t/main.swift'
```

Comment:
This is for `interpret` mode: I think `swift` command would build and run the script.